### ISSUE TITLE
Trouble installing using Mamba

### ISSUE THREAD

**Issue by cwinsnes** [kalininalab/DataSAIL#16]
## Issue description
I am having trouble installing `DataSAIL` through mamba. It is unable to find the package datasail because it "does not exist".

I run the command

`mamba create -n sail -c conda-forge -c kalininalab -c bioconda DataSAIL`

Which yields the following

```
[... output regarding channels ...]
Could not solve for environment specs
The following package could not be installed
└─ datasail does not exist (perhaps a typo or a missing channel).
```

The problem persists whether or not I run the create command with the additional `-c mosek` or not.

## System specifications
mamba 1.5.5  
conda 23.11.0  
python 3.10.13  

Running on a Debian Bullseye (Debian 11.8) Docker image, with a MacOS Sonoma (M2 chip) as the underlying system.

It is not possible to install the package on the MacOS system directly due to the package `cd-hit` being unavailable on such a system.

**Comment by Old-Shatterhand** [comment]
Hey @cwinsnes,

please excuse the late response.
The problem is that all clustering algorithms (mmseqs, cd-hit, foldseek, and mash) only have builds for osx and linux-64. Which is funny, because to my knowlegde, they can be installed from source on M2 chips. I'm working on a solution to provide datasail without the clustering algorithms (at least for win-64 and m2 chips.

In the mean time, there are two solutions: Either switch to osx (or disable the M2 chip somehow, there's a way to get this done, but I'm not a Mac-User so, I don't know how) or install datasail from source by cloning the repo, removing the problematic dependencies from the `meta.yaml` and building it locally.

**Comment by cwinsnes** [comment]
I thought working within the Docker container would alleviate that issue as it's Linux x64.

If compiling from source is the way, I'll try compiling that

**Comment by cwinsnes** [comment]
The problem unfortunately persists when I run the command with `python=3.10`. 

The version I used in the first post was Python 3.10.13, and I have tried with 3.11.2 as well, so it doesn't seem to me that it's the Python version.

**Comment by Old-Shatterhand** [comment]
This has been solved with commit [5fe4768](https://github.com/kalininalab/DataSAIL/commit/5fe4768c3d1f95a27166f0a30d843a107bcf0b5c) and in PR #22.
